consumer/push_consumer.go

```diff
@@ -216,7 +216,41 @@ func (pc *pushConsumer) IsUnitMode() bool {
 }
 
 func (pc *pushConsumer) messageQueueChanged(topic string, mqAll, mqDivided []*primitive.MessageQueue) {
-	// TODO
+	v, exit := pc.subscriptionDataTable.Load(topic)
+	if !exit {
+		return
+	}
+	data := v.(*internal.SubscriptionData)
+	newVersion := time.Now().UnixNano()
+	rlog.Infof("the MessageQueue changed, also update version: %d to %d", data.SubVersion, newVersion)
+	data.SubVersion = newVersion
+
+	// TODO: optimize
```

Comment:
IMO, it better to remove "TODO" here which make the developer confused, and you can create an issue to record that it needs polish in this file here.

Reply:
@wenfengwang

Reply:
issue created: https://github.com/apache/rocketmq-client-go/issues/175